feg/radius/src/run.sh

```diff
@@ -51,7 +51,8 @@ function pretty {
 }
 
 function test {
-    gotestsum ./...
+    go clean -testcache
```

Comment:
I assume in CI there are no caches to benefit from - but does this impact local execution times?

Reply:
I removed the line as caches will be ignored if there have been any changes. We used `go clean -testcache` previously to trigger flaky tests.